includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -356,6 +358,12 @@ public function getExploreMenu(): array {
 					'tracking' => 'explore-forum',
 					'include' => !empty( $wgEnableForumExt ) && !empty( $wgEnableDiscussions ),
 				],
+				[
+					'url' => 'www.amazon.com/fandom',
```

Comment:
Isn't that URL only for US? It links to US Amazon. Also if we want to check the country, we need probably to check the Geo cookie, so wg var may not work for us

Reply:
This was just a sample link to test the debug param.

Reply:
refactored and this no longer applies.